website/src/pages/documents/document.css.ts

```diff
@@ -224,3 +224,22 @@ export const mobileWordPanel = style([
     },
   },
 ])
+
+export const scrollTop = style({
+  display: "flex",
+  flexDirection: "row",
+  flexWrap: "wrap",
+  flexFlow: "column nowrap",
+  justifyContent: "flex-end",
+  alignContent: "center",
+  alignItems: "center",
+  alignSelf: "flex-end",
+  position: "sticky",
+  zIndex: 1,
+  top: `calc(100vh - 105px)`,
+  right: 0,
+  padding: `7px`,
```

Comment:
Use our constant values from the `style/constants` file like `hspace` and `vspace` for padding.